### DISCUSSION TITLE
New Feature: Item groups

### DISCUSSION THREAD

**Discussion by mrjones2014** [mrjones2014/legendary.nvim#228] [Show and tell]
Hello everyone,

Just merged item group support. Selecting an item group from the finder will open a new finder with that groups sub-items.

See [Quickstart](https://github.com/mrjones2014/legendary.nvim#quickstart).

![CleanShot-2022-12-03-at-21 26 40](https://user-images.githubusercontent.com/8648891/205471034-7b8cd04d-64e5-4de3-b913-c16209d9a984.gif)

**Comment by olimorris** [comment]
Slowly working through my config to make use of this amazing feature! The icons are a beautiful addition too
